plugins/main/bridge/bridge.go

```diff
@@ -386,25 +387,32 @@ func cmdDel(args *skel.CmdArgs) error {
 		return nil
 	}
 
+	// There is a netns so try to clean up. Delete can be called multiple times
+	// so don't return an error if the delete is already removed.
+	// If the device isn't there then don't try to clean up IP masq either.
 	var ipn *net.IPNet
 	err = ns.WithNetNSPath(args.Netns, func(_ ns.NetNS) error {
 		var err error
 		ipn, err = ip.DelLinkByNameAddr(args.IfName, netlink.FAMILY_V4)
+		if err != nil && strings.Contains(err.Error(), "Link not found") {
+			// Ignore the error if link isn't found.
```

Comment:
possibly unnecessary given apparent conformance to return error or value but not both paradigm of [`DelLinkByNameAddr`](https://github.com/containernetworking/cni/blob/master/pkg/ip/link.go#L168)